core/server.py

```diff
@@ -18,5 +24,62 @@ def load_poll(filename: str) -> Poll:
             content = file.read()
             return Poll.from_json(content)
 
+    @staticmethod
```

Comment:
znalazłem coś takiego w internecie:
`import socket
hostname = socket.gethostname()
IPAddr = socket.gethostbyname(hostname)

print("Your Computer Name is:" + hostname)
print("Your Computer IP Address is:" + IPAddr)
`
dlaczego nie możemy zrobić tego w powyższy sposób?

Reply:
bo na to nie wpadłem ale mogę spprawdzić jakie to daje efekty

Reply:
![image](https://github.com/OilyCannelloni/Software-Engineering-Backend/assets/92760953/5b05a73c-c4a2-4ca3-9a90-aa62259ea5df)

Reply:
także tak

Reply:
```
import socket

def get_local_ip():
    # Connect to a remote server to get the local IP address
    try:
        # The IP address used here (8.8.8.8) does not have to be reachable.
        # It's just used to determine the local IP address.
        s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
        s.connect(("8.8.8.8", 80))
        local_ip = s.getsockname()[0]
        s.close()
        return local_ip
    except Exception as e:
        return f"Error: {e}"

if __name__ == "__main__":
    ip = get_local_ip()
    print(f"Local IP address: {ip}")

```
ale takie coś już np działa